fiscal_epos_print/static/src/js/epson_epos_print.js

```diff
@@ -11,6 +13,109 @@ odoo.define("fiscal_epos_print.epson_epos_print", function (require) {
         return pad.substr(pad.length - padding, padding);
     }
 
+    function isErrorStatus(printerStatus) {
+        var error = false;
+        switch (printerStatus.substring(0, 2)) {
+            case "00":
+            case "01":
+            case "20":
+            case "21":
+                error = false;
+                break;
+            default:
+                error = true;
+        }
+        return error;
+    }
+
+    function decodeFpStatus(printerStatus) {
+        var printer = "";
+        var ej = "";
+        var receipt = "";
+
+        switch (printerStatus.substring(0, 1)) {
+            case "0":
+                printer = false;
+                break;
+            case "2":
+                printer = _t("Paper running low");
+                break;
+            case "3":
+                printer = _t("Offline (end of paper or open cover)");
+                break;
+            default:
+                printer = _t("Wrong answer");
+        }
+
+        switch (printerStatus.substring(1, 2)) {
+            case "0":
+                ej = false;
+                break;
+            case "1":
+                ej = _t("Running low");
+                break;
+            case "2":
+                ej = _t("To format");
+                break;
+            case "3":
+                ej = _t("Previous");
+                break;
+            case "4":
+                ej = _t("From other measurement device");
+                break;
+            case "5":
+                ej = _t("Finished");
+                break;
+            default:
+                ej = _t("Wrong answer");
+        }
+
+        switch (printerStatus.substring(3, 4)) {
+            case "0":
+                receipt = _t("Fiscal open");
+                break;
+            case "1":
+                receipt = false;
+                //receipt = "Fiscale/Non fiscale chiuso";
+                break;
+            case "2":
+                receipt = _t("Non fiscal open");
+                break;
+            case "3":
+                receipt = _t("Payment in progress");
+                break;
+            case "4":
+                receipt = _t("Error on last ESC/POS command with Fiscal/Non fiscal closed");
+                break;
+            case "5":
+                receipt = _t("Negative receipt");
+                break;
+            case "6":
+                receipt = _t("Error on last ESC/POS command with Non fiscal open");
+                break;
+            case "7":
+                receipt = _t("Waiting for receipt closing in JAVAPOS mode");
+                break;
+            case "8":
+                receipt = _t("Fiscal document open");
+                break;
+            case "A":
+                receipt = _t("Title open");
+                break;
+            case "2":
+                receipt = _t("Title closed");
+                break;
```

Comment:
il ```case "2"``` è già dichiarato alla righe 81-83, quale dei due è corretto?

Reply:
@GSLabIt :arrow_up: ?

Reply:
@eLBati dev'essere un refuso, da modificare in B.

Reply:
Corretto, grazie